LICENSE

```diff
@@ -1,7 +1,4 @@
-Copyright (c) 2012 Vladimir Keleshev <vladimir@keleshev.com>
-                   Blake Williams <code@shabbyrobe.org>
-                   Alex Speller <alex@alexspeller.com>
-                   Nima Johari
+Copyright (c) 2020 Eyal Shalev
 
```

Comment:
Not sure about the copyright information because the logic was shamelessly copied from [docopt.rb](https://github.com/docopt/docopt.rb)

Reply:
Maybe
```
Copyright (c) 2012 Vladimir Keleshev <vladimir@keleshev.com>
                   Eyal Shalev <eyalsh@gmail.com>
                   Blake Williams <code@shabbyrobe.org>	
                   Alex Speller <alex@alexspeller.com>	
                   Nima Johari	
```

Reply:
good point